rcl/test/rcl/client_fixture.cpp

```diff
@@ -24,6 +24,33 @@
 #include "../memory_tools/memory_tools.hpp"
 #include "../scope_exit.hpp"
 #include "rcl/error_handling.h"
+#include "rcl/graph.h"
+
+bool
+wait_for_server_to_be_available(
```

Comment:
I'd recommend the name `wait_for_service_to_be_available`, as `wait_for_server_to_be_available` is a bit ambiguous and `wait_for_service_server_to_be_available`, while explicit, is quite tedious.